packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -307,11 +317,16 @@ class Navigation extends PureComponent {
 
         return (
             <aside className={className}>
-                <div className={classes.header}>
+                <div
+                    className={`${classes.header} ${
+                        isCategoryRoot ? classes.isRoot : null
+                    }`}
```

Comment:
Same feedback as above. Please build up this value in a variable above the `return` statement and use it here.

Reply:
` const isRootClass = isCategoryRoot ? classes.isRoot : null;`

@supernova-at : I have performed suggested changes, please help to have a look into.